python/python/lance/dataset.py

```diff
@@ -975,18 +976,14 @@ def with_fragments(
 
     def nearest(
         self,
-        column: Optional[str] = None,
-        q: Optional[pa.FloatingPointArray] = None,
+        column: str,
```

Comment:
For some reason I thought `column` could default to "vector" but maybe that is applied elsewhere?  It's also not really an intuitive default so I wouldn't mind if we did away with it.

Reply:
That's applied at LanceDB level.